src/Numerics/Optimization/BfgsBMinimizer.cs

```diff
@@ -55,9 +92,9 @@ public MinimizationResult FindMinimum(IObjectiveFunction objective, Vector<doubl
 
             // Determine active set
             var gradientProjectionResult = QuadraticGradientProjectionSearch.Search(objective.Point, objective.Gradient, pseudoHessian, lowerBound, upperBound);
-            var cauchyPoint = gradientProjectionResult.Item1;
-            var fixedCount = gradientProjectionResult.Item2;
-            var isFixed = gradientProjectionResult.Item3;
+            var cauchyPoint = gradientProjectionResult.CauchyPoint;
+            var fixedCount = gradientProjectionResult.FixedCount;
+            var isFixed = gradientProjectionResult.IsFixed;
             var freeCount = lowerBound.Count - fixedCount;
 
             if (freeCount > 0)
```

Comment:
Code coverage suggests that `freeCount` is never larger than 0. Experimented with test that would solve cover this path but didn't succeed.